### PR TITLE
Fix padding for leaf nodes

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix](https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix)

File: src/app/public/modules/angular-tree/angular-tree-wrapper.component.scss [hunk 1]
```diff
@@ -70,7 +70,7 @@
   }
 
   .sky-switch-control {
-    margin: 0 $sky-margin 0 0;
+    margin: 0 8px 0 0;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/angular-tree/angular-tree-wrapper.component.scss:73]: ☝️ The width of our checkboxes doesn't match the width of our caret icons, so unfortunately this couldn't be a nice round number.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix](https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix)

**Review by Blackbaud-ToddRoberts — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix](https://developer.blackbaud.com/skyux-angular-tree-component?_sp.spa=pr-spacing-fix)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Pull request merged**